app/assets/javascripts/discourse-markdown-it/src/features/image-controls.js

```diff
@@ -121,13 +122,14 @@ function buildImageEditAltTextControls(altText) {
 
 function buildImageDeleteButton() {
   return `
-  <span class="delete-image-button" aria-label="${I18n.t(
-    "composer.delete_image_button"
-  )}">
-  <svg class="fa d-icon d-icon-trash-alt svg-icon svg-string" xmlns="http://www.w3.org/2000/svg">
-  <use href="#far-trash-alt"></use>
-  </svg>
-   </span>
+  <span class="delete-image-button" 
+    title="${I18n.t("composer.delete_image_button")}" 
+    aria-label="${I18n.t("composer.delete_image_button")}"
```

Comment:
I _think_ you don't need `aria-label` if there's a `title`?

```suggestion
    title="${I18n.t("composer.delete_image_button")}" 
```